Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "eppo"
+version = "0.1.0"
```

Comment:
Good for now - will bump to `1.0.0` when ready for release. Is that appropriate convention in rust?

Reply:
I think we want to publish this as version 3.0.0 once fully ready to indicate that this version of the SDK is using the UFC